src/WiFiClient.cpp

```diff
@@ -63,8 +63,7 @@ int WiFiClient::connect(IPAddress ip, uint16_t port) {
 
     	unsigned long start = millis();
 
-    	// wait 4 second for the connection to close
```

Comment:
I removed these comments as they seem to be completely unrelated. (Copy paste accident from earlier?)

Reply:
it may be used later on

Reply:
But the comment was completely wrong. It was waiting to connect, not for the connection to close, and it waited 10 seconds not 4...

Reply:
Okay